showItemPageBE/src/main/java/com/talentpath/shamazin/showItemPage/models/Item.java

```diff
@@ -21,6 +21,9 @@
     @OneToMany(mappedBy="item", cascade=CascadeType.ALL)
     private List<ProductPhoto> productPhotos;
 
+    @OneToMany(mappedBy = "question", cascade=CascadeType.ALL)
+    private List<Question> questions;
```

Comment:
Hey Matt, just checking, I thought you said you were linking questions to ItemFamily. Let me know if you changed you mind